lib/iris/fileformats/pp_load_rules.py

```diff
@@ -1155,6 +1155,27 @@ def _all_other_rules(f):
             )
         )
 
+    if (
+        f.bdx != 0.0
+        and f.bdx != f.bmdi
+        and len(f.lbcode) != 5
+        and f.lbcode[0] == 9
+    ):
+        dim_coords_and_dims.append(
+            (
+                DimCoord.from_regular(
+                    f.bzx,
+                    f.bdx,
+                    f.lbnpt,
+                    standard_name="projection_x_coordinate",
+                    units="metres",
+                    circular=(f.lbhem in [0, 4]),
+                    coord_system=None,
```

Comment:
I'm unsure about this. I was wondering if this should be a type of Mercator projection?

Reply:
I think if it were Mercator we'd use a different code? The spec specifies Mercator as different. We'd also need a globe underlying the Mercator CS I think, which wouldn't make sense for a plane?

Will check with the user, though they seemed to be mostly relying on the workaround we gave them in the first place rather than sure what the correct answer was

Reply:
I see you're right about the lbcode for a mercator coord system. Although I was more trying to refer to the family of mercator (so also including transverse mercator, pseudo mercator, utm, and maybe some more that I'm not familiar with?) as those would have been my best guess for what the coord system would be, but as PP has an encoding for Mercator my guess seems unlikely!

I'm a bit uncomfortable to set the coord_system as None. Coordinates are mostly meaningless unless you have that information. Iris also makes some interesting assumptions when a coord_system is missing, for example in plotting, if no coord_system is present, it will assume PlateCarree:
https://github.com/SciTools/iris/blob/main/lib/iris/plot.py#L870-L874
 
I'm also a little concerned that in their earlier emails, the user mentions "Note the Cartesian grid can be wrap around in both X and Y," and mentions lbhem as the determining factor, but I can't see how you could interpret when int wraps in both X and Y and when it wraps only x and not Y. Also I'm not so sure what how extensive Iris' handling of non X-like circular coordinates is

Reply:
I think because we're dealing with idealised models (according to the user) there just isn't a projection.

I think all Iris coordinate systems (and at least all that the pp loader can load) are Geographic Coordinate Systems (and thus can't be about a flat surface). Which seems to constrain us to a) leave None there, b) put in an incorrect coordinate system, c) don't support LBCODE 9?

I can get more information on how LBHEM relates to x wrapping and y wrapping, which can inform us on how this should be written if we implement it.

In terms of whether the support is there in the rest of Iris to handle this, it should probably either refuse or try its best when asked to plot it, unless we go through everything building this case in. I've asked for information from the user about how much it's used.